src/components/SingleDay/SingleDay.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState, type FunctionComponent } from 'react';
+import { getCurrentDate, saveNewDay } from '../../utils/utils';
+import SquareButton from '../ui/Buttons/SquareButton';
+import RectangleButton from '../ui/Buttons/RectangleButton';
+import TextArea from '../ui/Inputs/TextArea';
+
+const SingleDay: FunctionComponent = () => {
+  const [dayText, setDayText] = useState('');
+  const [isTextAreaEmpty, setIsTextAreaEmpty] = useState(true);
+  const date = getCurrentDate();
+
+  function textAreaHandler(textAreaValue: string): void {
+    setDayText(textAreaValue);
+
+    if (textAreaValue.trim().length) {
+      setIsTextAreaEmpty(false);
+    } else {
+      setIsTextAreaEmpty(true);
+    }
```

Comment:
```suggestion
   setIsTextAreaEmpty(!textAreaValue.trim().length);
```
аналогично твоей записи, но более лаконично

Reply:
если записиь через ! тебе кажется не очень читаемой можно заменить на Boolean(...). такие вещи обычно решаются на уровне компании и записываются в eslint правила